### DISCUSSION TITLE
Welcome to Sequel-Ace Discussions!

### DISCUSSION THREAD

**Discussion by Kaspik** [Sequel-Ace/Sequel-Ace#603] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tombajzek** [comment]
I've been building web applications since about 2000, and have use Sequel Pro, and now Sequel Ace for database management this entire time. Recently, I've been experiencing cases where launching Sequel Ace results in the app coming up frozen. I've submitted this through the macOS Crash Reporter, but it appears this has not reached anyone on the development team. Resolving this usually requires me to re-install the app, but sometimes I need to find and delete the support files in the Library. I'm in that situation now, with version 4.0.13 on my M1 MacBook Air.
I have an older Intel Mac running 4.0.7 that does not have this issue. I download the app from the App Store, not from Github.

Has anyone else seen this? Does anyone have any remedies for this?

**Comment by Fajis** [comment]
hi i really loved the app better than most of the apps i used so far , it would really help if i had an option for search in database list i will be glad if it came in future releases thanks in advance
